x-pack/plugins/observability/public/pages/alerts/index.tsx

```diff
@@ -60,17 +62,41 @@ export function AlertsPage({ routeParams }: AlertsPageProps) {
   // observability. For now link to the settings page.
   const manageRulesHref = prepend('/app/management/insightsAndAlerting/triggersActions/alerts');
 
-  const { data: dynamicIndexPatternResp } = useFetcher(({ signal }) => {
+  const { data: indexNames = NO_INDEX_NAMES } = useFetcher(({ signal }) => {
     return callObservabilityApi({
       signal,
       endpoint: 'GET /api/observability/rules/alerts/dynamic_index_pattern',
+      params: {
+        query: {
+          namespace: 'default',
+          registrationContexts: [
+            'observability.apm',
+            'observability.logs',
+            'observability.infrastructure',
```

Comment:
`observability.infrastructure` registration context doesn't exist in the code. What plugin is supposed to be defining it?

Reply:
I think this should be `observability.metrics`

Reply:
TODO in the follow-up PR: remove `observability.infrastructure` and make sure `observability.metrics` is there.